transport/internet/http/hub.go

```diff
@@ -104,9 +107,35 @@ func Listen(ctx context.Context, address net.Address, port net.Port, streamSetti
 
 	config := tls.ConfigFromStreamSettings(streamSettings)
 	if config == nil {
-		return nil, newError("TLS must be enabled for http transport.").AtWarning()
-	}
+		// return nil, newError("TLS must be enabled for http transport.").AtWarning()
```

Comment:
H2C has been sent separately, I suggest you re-initiate PR

Reply:
You are right, I'll re-initiate PR.